src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  display: block;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  border-radius: 0.5px;
```

Comment:
Not sure we need border-radius in this task.

Reply:
It was in figma styles so i added it without questioning :D